READme.md

```diff
@@ -0,0 +1,187 @@
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Responsive-Design**
+- **Call-to-action**
+- **Hero-Section**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Will be available soon.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+
+In order to run this project you need:
+
+A web browser that supports HTML and CSS.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone git@github.com:myaccount/Portfolio.git
+
+
+### Usage
+Open the cloned project folder and navigate to the index.html file. Double-click on the file to open it in your web browser. This will display the project's HTML and CSS content.
+
+
+### Customization
+To customize the project, you can modify the HTML and CSS files according to your preferences. Open the project folder in a text editor of your choice, such as Visual Studio Code, and make the desired changes.
+
+
+### Run tests
+
+To run tests, run the following command:
+
+Open in chrome.
+
+### Deployment
+
+To deploy this project, you can upload the project folder (including the HTML and CSS files) to a web server or hosting service of your choice. Ensure that the server supports serving static HTML and CSS files.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@Xghayor](https://github.com/Xghayor)
+- LinkedIn: [Ghayoorulbaqir](https://linkedin.com/in/ghayoor-ul-baqir)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> 
+
+- [ ] **Achievements**
+- [ ] **Projects**
+- [ ] **Contact-form**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> If you like this project and find it useful, I would greatly appreciate your support! By contributing to this project, you can help it grow and improve for the benefit of others. There are several ways you can support:
+
+1.Spread the word
+2.Contribute
+3.Provide feedback
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+
+> I would like to thank Microvers.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Kindly add a License to your project as the `Mit` link on your readme currently is broken to have a professional readme